website/content/docs/agent/options.mdx

```diff
@@ -1482,7 +1482,7 @@ bind_addr = "{{ GetPrivateInterfaces | include \"network\" \"10.0.0.0/8\" | attr
 - `enable_agent_tls_for_checks` When set, uses a subset of the agent's TLS configuration (`key_file`,
   `cert_file`, `ca_file`, `ca_path`, and `server_name`) to set up the client for HTTP or gRPC health checks. This allows services requiring 2-way TLS to be checked using the agent's credentials. This was added in Consul 1.0.1 and defaults to false.
 
-- `enable_central_service_config` When set, the Consul agent will look for any centralized service
+- `enable_central_service_config` When set, the Consul agent will look for any [centralized service](https://www.consul.io/docs/agent/config-entries) 
```

Comment:
Oh, I think this link should be `/docs/agent/config-entries`. I think generally we try to use relative links when possible.

It might also make sense to include the "configurations" in the link:

```
[centralized service configurations](/docs/agent/config-entries)
````